.github/workflows/actions/auto-request-reviews/main.js

```diff
@@ -0,0 +1,45 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+
+async function run() {
+  const token = core.getInput('token');
+  const octokit = github.getOctokit(token);
+
+  const organization = core.getInput('org');
+  const teamSlug = core.getInput('team');
```

Comment:
Do these lookups fail at runtime if they're not provided?

Reply:
Turns out they don't fail (which seemed weird to me 🤷‍♂️ ). So I added some manually checks that will throw errors in those cases